Docs/MmSupervisorDesign.md

```diff
@@ -0,0 +1,171 @@
+# Microsoft MM Supervisor Overview
+
+## Overview
+
+The Microsoft MM Supervisor implements a SMM isolation solution that meets the
+[Secured-core PC requirements](https://learn.microsoft.com/windows-hardware/design/device-experiences/oem-highly-secure-11).
+
+The MM Supervisor combines the latest best practices for SMM safety including:
+
+* A customizable security policy that can be tailored to the platform
+* Early boot (PEI) launch capability
+* Privilege separation between supervisor and user mode
+* Stringent memory isolation
+
+Key features of the MM Supervisor include:
+
+* The MM foundation is initialized early in the boot process and has isolated protocol and configuration table database
+* The MM supervisor holds separate service databases for supervisor and user mode services
+* Platform MM drivers and handlers execute under user mode (CPL3)
+* It follows a binary release model

```

Comment:
Currently this isn't true.

Reply:
I see in a different document this is listed as one of our implementation goals.  For that reason I'm closing this comment.